README.md

```diff
@@ -64,7 +66,7 @@ adyen.api_key = 'AF5XXXXXXXXXXXXXXXXXXXX'
 
 - Make a Payment
 ~~~~ruby
-response = adyen.checkout.payments({
+response = adyen.checkout.payments_api.payments({
```

Comment:
Do we need to add client setup to this example?

Reply:
Hmmm think it's pretty clear as the client setup is right above it